exporters/otlp/recordable_test.cc

```diff
@@ -0,0 +1,64 @@
+#include "exporters/otlp/recordable.h"
+
+#include <gtest/gtest.h>
+
+using namespace opentelemetry::exporter::otlp;
```

Comment:
Instead of using declaration for entire namespace, it would be better to just put all tests in the same namespaces:

namespace opentelemetry {
namespace exporter {
namespace otlp {

This would also remove the need to specify opentelemetry namespace for things like trace::TraceId.